lib/utils.js

```diff
@@ -7,7 +7,7 @@ const parser = require('@asyncapi/parser');
  * @returns {Array} parsed AsyncAPI documents
  */
 function validate(asyncApiDocs) {
-  return Promise.all(asyncApiDocs.map(async doc => parser.parse(doc.json())));
+  return Promise.all(asyncApiDocs.map(async doc => parser.parse(doc._json)));
```

Comment:
`x-parser-spec-parsed` is a extension which indicates that object is a parsed AsyncAPI doc.

Reply:
side feeling :) I think we definitely should wrap this extension in a function in a parser. Something for the future, to not access props like that

Reply:
> I think that `.json()` function is better than normal `_json`. We shouldn't use the "private" properties from parser.

Initially, I had `.json()` but on running the tests, this error keeps generating that `.json()` is not a function.

Reply:
> Initially, I had .json() but on running the tests, this error keeps generating that .json() is not a function.

Because you probably operate on string, not on parsed doc :)